src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -40,6 +40,21 @@ public SyndicationFeed Feed
             }
         }
 
+        public Func<string, UriKind, string, string, Uri> UriParser { get; set; } = DefaultUriParser;
+
+        // Different DateTimeParsers are needed for Atom and Rss so can't set inline
+        public Func<string, string, string, DateTimeOffset> DateTimeParser { get; set; }
+
+        internal virtual Func<string, string, string, DateTimeOffset> GetDefaultDateTimeParser()
+        {
+            return NotImplementedDateTimeParser;
+        }
+
+        private DateTimeOffset NotImplementedDateTimeParser(string dtoString, string localName, string ns)
+        {
```

Comment:
This will still cause a derived class to throw where it didn't throw before. This gives the requirement that code which uses a derived class must set DateTimeParser.